internal/provider/provider.go

```diff
@@ -210,12 +219,20 @@ func (p *IncusProvider) Configure(ctx context.Context, req provider.ConfigureReq
 	// }
 	envName := os.Getenv("INCUS_REMOTE")
```

Comment:
Since we enforce now that we have a default remote, this code should be changed and drop the environment variables.